LuhnAlgorithm/.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think that this file has nothing to do with commited functionality. Try to make commits as granular as possible including only the functional changes. 
Also, you could add a `.gitignore` file in the root of your project. if you don't have it, and add the intelliJ folder there (`.idea`), it will prevent commiting of these files/folders